test/test_fake_quant.py

```diff
@@ -142,5 +175,119 @@ def test_fq_serializable(self):
         self.assertEqual(loaded_fq_module.calculate_qparams(), fq_module.calculate_qparams())
 
 
+class TestFakeQuantizePerChannel(TestCase):
```

Comment:
Is it possible to merge this with TestFakeQuantizePerTensor rather than copy paste the code?

Reply:
The problem is with the hypothesis decorator outside the function. In one case it is a regular tensor and in another a per-channel tensor, not sure how to merge this.

Reply:
I think we can use draw to achieve this: https://hypothesis.readthedocs.io/en/latest/data.html#drawing-interactively-in-tests, but we can do this in a separate PR.